deploy/helm/kubecf/templates/brain-tests.yaml

```diff
@@ -0,0 +1,223 @@
+# psp-role       | predefined ? -> scf default account
+# configgin-role | predefined ? -> scf default account
+
+# Service account, roles, and cluster role for the brain tests.
+# Raw kube objects going into the chart.
+# TODO: Make this conditional.
+
+# Service account "tests-brain" is used by the instance group "acceptance-tests-brain"
+---
+apiVersion: "v1"
+kind: "ServiceAccount"
+metadata:
+  name: "{{ .Values.deployment_name }}-tests-brain"
+  namespace: {{ .Release.Namespace | quote }}
+  labels:
+    app.kubernetes.io/component: "{{ .Values.deployment_name }}-tests-brain"
+    app.kubernetes.io/instance: {{ .Release.Name | quote }}
+    app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+    app.kubernetes.io/name: {{ include "kubecf.fullname" . }}
+    app.kubernetes.io/version: {{ default .Chart.Version .Chart.AppVersion | quote }}
+    helm.sh/chart: {{ include "kubecf.chart" . }}
+
+# Role "test-role-brain" only used by account "tests-brain"
+# The permissions are a combination of the SCF roles
+# configgin-role, psp-role, and test-brain-role.
+---
+apiVersion: "rbac.authorization.k8s.io/v1"
+kind: "Role"
+metadata:
+  name: "{{ .Values.deployment_name }}-test-role-brain"
+  labels:
+    app.kubernetes.io/component: "{{ .Values.deployment_name }}-test-role-brain"
+    app.kubernetes.io/instance: {{ .Release.Name | quote }}
+    app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+    app.kubernetes.io/name: {{ include "kubecf.fullname" . }}
+    app.kubernetes.io/version: {{ default .Chart.Version .Chart.AppVersion | quote }}
+    helm.sh/chart: {{ include "kubecf.chart" . }}
+rules:
```

Comment:
We should, at some later time, go through each test and explicitly note down the relationship between permissions and tests, so that we might eventually be able to drop some permissions.  File a new issue for that, perhaps?

Reply:
As I noted at the end of the PR description, explaining why I ticked the security implication box.

Reply:
New ticket is https://github.com/SUSE/kubecf/issues/227